packages/create-plugin/templates/common/_package.json

```diff
@@ -51,6 +51,7 @@
     "sass-loader": "13.3.1",
     "style-loader": "3.3.3",
     "swc-loader": "^0.2.3",
+    "terser-webpack-plugin": "^5.3.10",
```

Comment:
We need this updated version of terser to be able to make fine grained choices of which `console.*` methods to drop.